src/morse/index.js

```diff
@@ -58,6 +58,28 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  if (typeof text !== "string") {
+    throw "Please provide a morse string";
```

Comment:
Throw `Error` rather than string like you did in `isolateDuplicates`. Doing that preserves stack trace.